CocoaPods/Podfile

```diff
@@ -12,8 +12,6 @@ plugin 'cocoapods-art', :sources => [
 ]
 
 target 'TomTomSDKExamples' do
-  pod 'TomTomSDKCommon', '0.32.0'
```

Comment:
These libraries are mentioned in the "[Building a navigation app](https://developer.tomtom.com/ios/navigation/documentation/use-cases/building-a-navigation-app)" use case in the developer portal.

Reply:
https://github.com/tomtom-international/tomtom-navigation-ios-examples/pull/47#discussion_r1399314516